daemon/image_delete.go

```diff
@@ -132,15 +146,16 @@ func (daemon *Daemon) imgDeleteHelper(name string, list *[]types.ImageDelete, fi
 				if first {
 					return err
 				}
-
 			}
-
 		}
 	}
 	return nil
 }
 
 func (daemon *Daemon) canDeleteImage(imgID string, force bool) error {
+	directlyUsedBy := []string{}
```

Comment:
Transitive: If we have a graph `A -> B -> C`, we say that `A` _depends_ on `B` and `C`. More granularly, `A` _directly depends_ on `B` but `A` _transitively depends_ on `C`.

@dmcgowan PTAL